test/unit/transform/ferc1_test.py

```diff
@@ -1 +1,84 @@
 """Unit tests specific to the FERC Form 1 table transformations."""
+
+from io import StringIO
+
+import pandas as pd
+import pytest
+
+from pudl.settings import Ferc1Settings
+from pudl.transform.ferc1 import fill_dbf_to_xbrl_map, read_dbf_to_xbrl_map
+
+TEST_DBF_XBRL_MAP = pd.read_csv(
+    StringIO(
+        """
+sched_column_name,report_year,row_literal,row_number,row_type,xbrl_column_stem
+test_table1,2000,"Header 1",1,header,"N/A"
+test_table1,2000,"Account A",2,ferc_account,account_a
+test_table1,2000,"Account B",3,ferc_account,account_b
+test_table1,2000,"Header 2",4,header,"N/A"
+test_table1,2000,"Account C",5,ferc_account,account_c
+test_table1,2002,"Header 1",1,header,"N/A"
+test_table1,2002,"Account A",2,ferc_account,account_a
+test_table1,2002,"Account B",3,ferc_account,account_b
+test_table1,2002,"Account B1",4,ferc_account,account_b1
+test_table1,2002,"Header 2",5,header,"N/A"
+test_table1,2002,"Account C",6,ferc_account,account_c
+"""
+    ),
+)
+
+
+@pytest.mark.parametrize(
+    "dbf_table_name",
+    [
+        "f1_plant_in_srvce",
+        "f1_elctrc_erg_acct",
+    ],
+)
+def test_actual_dbf_to_xbrl_maps(dbf_table_name):
+    """Test our DBF to XBRL row alignment tools & manually compiled mapping."""
+    dbf_xbrl_map = fill_dbf_to_xbrl_map(
+        df=read_dbf_to_xbrl_map(dbf_table_name=dbf_table_name),
+        dbf_years=Ferc1Settings().dbf_years,
+    )
+    dbf_to_xbrl_mapping_is_unique = (
+        dbf_xbrl_map.groupby(["report_year", "xbrl_column_stem"])[
+            "row_number"
+        ].nunique()
+        <= 1
+    ).all()
+
+    assert dbf_to_xbrl_mapping_is_unique  # nosec: B101
```

Comment:
can you make this an assertion error with a message? this fail would be real cryptic if you weren't you

Reply:
I made te test name and docstring more specific to what's actually being tested.